base/rational.jl

```diff
@@ -279,6 +279,11 @@ function *(x::Rational, y::Rational)
     xd,yn = divgcd(x.den,y.num)
     checked_mul(xn,yn) // checked_mul(xd,yd)
 end
+function *(x::Rational, y::Integer)
+    xd, yn = divgcd(x.den, y)
+    checked_mul(x.num,yn) // xd
```

Comment:
```suggestion
    checked_mul(x.num, yn) // xd
```

Reply:
Thanks for cleaning up my misguided attempt to 'mirror surrounding syntax'!